quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/SuspendablesScanner.java

```diff
@@ -715,7 +714,7 @@ void setSupers(String superName, String[] interfaces) {
         }
 
         void setMethods(Collection<String> ms) {
-            this.methods = ms.toArray(new String[ms.size()]);
+            this.methods = ms.toArray(new String[0]);
```

Comment:
This is confusing. Are you really sure about this?

Reply:
Yes, this is the latest advice with modern JVMs.